README.md

```diff
@@ -1,160 +1,132 @@
-    "30": 0.01,   # Sell after 30 minutes if there is at least 1% profit
-    "20": 0.02,   # Sell after 20 minutes if there is at least 2% profit
-    "0":  0.04    # Sell immediately if there is at least 4% profit
-},
-```
-
-`stoploss` is loss in percentage that should trigger a sale.
-For example value `-0.10` will cause immediate sell if the
-profit dips below -10% for a given trade. This parameter is optional.
-
-`initial_state` is an optional field that defines the initial application state.
-Possible values are `running` or `stopped`. (default=`running`)
-If the value is `stopped` the bot has to be started with `/start` first.
-
-`ask_last_balance` sets the bidding price. Value `0.0` will use `ask` price, `1.0` will
-use the `last` price and values between those interpolate between ask and last
-price. Using `ask` price will guarantee quick success in bid, but bot will also
-end up paying more then would probably have been necessary.
-
-`fiat_display_currency` set the fiat to use for the conversion form coin to 
-fiat in Telegram. The valid value are: "AUD", "BRL", "CAD", "CHF", 
-"CLP", "CNY", "CZK", "DKK", "EUR", "GBP", "HKD", "HUF", "IDR", "ILS", 
-"INR", "JPY", "KRW", "MXN", "MYR", "NOK", "NZD", "PHP", "PKR", "PLN",
-"RUB", "SEK", "SGD", "THB", "TRY", "TWD", "ZAR", "USD".
-
-The other values should be self-explanatory,
-if not feel free to raise a github issue.
-
-### Prerequisites
-* python3.6
-* sqlite
-* [TA-lib](https://github.com/mrjbq7/ta-lib#dependencies) binaries
-* Minimal (advised) system requirements: 2GB RAM, 1GB data, 2vCPU
-
-### Install
-
-#### Arch Linux
-
-Use your favorite AUR helper and install `python-freqtrade-git`.
-
-#### Manually
-
-`master` branch contains the latest stable release.
-
-`develop` branch has often new features, but might also cause breaking changes. To use it, you are encouraged to join our [slack channel](https://join.slack.com/t/highfrequencybot/shared_invite/enQtMjQ5NTM0OTYzMzY3LWMxYzE3M2MxNDdjMGM3ZTYwNzFjMGIwZGRjNTc3ZGU3MGE3NzdmZGMwNmU3NDM5ZTNmM2Y3NjRiNzk4NmM4OGE).
-
-```
-$ cd freqtrade/
-# copy example config. Dont forget to insert your api keys
-$ cp config.json.example config.json
-$ python -m venv .env
-$ source .env/bin/activate
-$ pip install -r requirements.txt
-$ pip install -e .
-$ ./freqtrade/main.py
+[![Coverage Status](https://coveralls.io/repos/github/gcarq/freqtrade/badge.svg?branch=develop&service=github)](https://coveralls.io/github/gcarq/freqtrade?branch=develop)
+
+
+Simple High frequency trading bot for crypto currencies designed to 
+support multi exchanges and be controlled via Telegram.
+
+![freqtrade](https://raw.githubusercontent.com/gcarq/freqtrade/develop/docs/assets/freqtrade-screenshot.png)
+
+## Disclaimer
+This software is for educational purposes only. Do not risk money which 
+you are afraid to lose. Always start by running a trading bot in Dry-run and do not engage money
+before you understand how it works and what profit/loss you should
+expect.
```

Comment:
I wonder if we should use this opportunity to add a final note along the lines of USE THE SOFTWARE AT YOUR OWN RISK. THE AUTHORS AND ALL AFFILIATES ASSUME NO RESPONSIBILITY FOR YOUR TRADING RESULTS.

Reply:
I really like your additions here though.

Reply:
I like it. Added to the documentation.